### PR TITLE
M sc1 61 19

### PR DESCRIPTION
**Pull request by GRAFOLAF**
Could you view my home task
Thx. =)

### CODE REVIEW

**Review by GRAFOLAF — COMMENTED** [review]
Could you view my home task, thx =) (HTML_02)

File: Nmaksimchicov/3.HTML/1.html [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+			<input type="text" name="search">
+			<input type="submit" value="Загрузить">
+		</fieldset>
+	</form>
+	<h2>Валидация данных HTML5:</h2>
+	<form method="post" enctype="http://sc.it-academy.by/fe/form.php">
+		<fieldset>
+			<legend>Строка поиска</legend>
+			<input type="text" placeholder="Только русские буквы">
+			<label form="name"></label><em>Телефонный номер в формате +ХХХ-ХХ-ХХХХХХХ, где Х - любое число</em>
```
Comment (pluhin) [inline Nmaksimchicov/3.HTML/1.html]: empty label

File: Nmaksimchicov/3.HTML/1.html [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+			<input type="text" placeholder="введите номер">
```
Comment (pluhin) [inline Nmaksimchicov/3.HTML/1.html]: missing pattern, name and id for label

File: Nmaksimchicov/3.HTML/1.html [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+			<label form="name"></label>Срок хранения(года):
+			<input type="text" name="search">
+			<input type="submit" value="Загрузить">
+		</fieldset>
+	</form>
+	<h2>Валидация данных HTML5:</h2>
+	<form method="post" enctype="http://sc.it-academy.by/fe/form.php">
+		<fieldset>
+			<legend>Строка поиска</legend>
+			<input type="text" placeholder="Только русские буквы">
```
Comment (pluhin) [inline Nmaksimchicov/3.HTML/1.html]: missing pattern, name

File: Nmaksimchicov/3.HTML/1.html [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+				</optgroup>
+			</select>
+			<input type="submit" value="OK">
+		</fieldset>
+	</form>
+	<h2>Загрузка файла на сервер:</h2>
+	<form method="post" enctype="multipart/form-data" action="http://sc.it-academy.by/fe/form.php">
+		<fieldset>
+			<legend>Upload</legend>
+			<label form="name"></label> Файл:
```
Comment (pluhin) [inline Nmaksimchicov/3.HTML/1.html]: empty label

File: Nmaksimchicov/3.HTML/1.html [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+			<input type="file" name="file_Field" 
+			dir="ltr">
+			<label form="name"></label>Срок хранения(года):
```
Comment (pluhin) [inline Nmaksimchicov/3.HTML/1.html]: empty label

File: Nmaksimchicov/3.HTML/1.html [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+		<input type="text" id="name" name="search" required autofocus><span>*</span>
+		<label form="e-mail">e-mail</label><span>*</span>
+		<input type="e-mail" id="e-mail" name="search" required autofocus><span>*</span>
+		<textarea name="text_area" lang="ru">Текст по умолчанию</textarea>
+		<input type="submit" value="Отправить"><br>
+	</fieldset>	
+	</form>
+</form>
+	<h2>Выпадающий и мульти списки:</h2>
+	<form method="get" action="http://sc.it-academy.by/fe/form.php">
```
Comment (pluhin) [inline Nmaksimchicov/3.HTML/1.html:103]: here just one list, should be two

File: Nmaksimchicov/3.HTML/1.html [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+		</label>
+	</fieldset>
+</form>
+<form>
+	<h2>Отзыв:</h2>
+	<form method="post" action="http://sc.it-academy.by/fe/form.php">
+	<fieldset>
+		<legend>Оставьте ваш отзыв:</legend>
+		<label form="name">Имя</label>
+		<input type="text" id="name" name="search" required autofocus><span>*</span>
```
Comment (pluhin) [inline Nmaksimchicov/3.HTML/1.html]: too many autofocuses on page

File: Nmaksimchicov/3.HTML/1.html [hunk 8]
```diff
@@ -0,0 +1,145 @@
...
+		</label>
+	</fieldset>
+</form>
+</form>
+<form>
+<h2>Опрос(Голосоывние):</h2>
+<form method="post" action="http://sc.it-academy.by/fe/form.php">
+	<fieldset>
+		<legend>Как вам?</legend>
+		<label>
```
Comment (pluhin) [inline Nmaksimchicov/3.HTML/1.html]: need remove this label - it is extra

File: Nmaksimchicov/3.HTML/1.html [hunk 9]
```diff
@@ -0,0 +1,145 @@
...
+<form>
+<h2>Простая форма авторизации</h2>
+<form method="post" action="http://sc.it-academy.by/fe/form.php">
+	<fieldset>
+		<legend>Авторизируйтесь</legend>
+		<label form="login">Логин</label>
+		<input type="text" id="login"name="login" required autofocus><span>*</span>
+		<label form="pass">Пароль:</label>
+		<input type="password" id="pass"name="password" required autofocus><span>*</span>
+		<label>
```
Comment (pluhin) [inline Nmaksimchicov/3.HTML/1.html:58]: one label for two inputs? It is some kind of time-economy?

File: Nmaksimchicov/3.HTML/1.html [hunk 10]
```diff
@@ -0,0 +1,145 @@
...
+		</label><br>
+	</fieldset>
+</form>
+<form>
+<h2>Простая форма авторизации</h2>
+<form method="post" action="http://sc.it-academy.by/fe/form.php">
+	<fieldset>
+		<legend>Авторизируйтесь</legend>
+		<label form="login">Логин</label>
+		<input type="text" id="login"name="login" required autofocus><span>*</span>
```
Comment (pluhin) [inline Nmaksimchicov/3.HTML/1.html]: autofocus should be one on page

File: Nmaksimchicov/3.HTML/1.html [hunk 11]
```diff
@@ -0,0 +1,145 @@
...
+			<input type="radio" name="search_dist" value="site">
+			на сайте
+		</label>
+		<label>
+			<input type="radio" name="search_dist" value="google" checked>
+			в Google
+		</label><br>
+	</fieldset>
+</form>
+<form>
```
Comment (pluhin) [inline Nmaksimchicov/3.HTML/1.html]: what is form here?

File: Nmaksimchicov/3.HTML/1.html [hunk 12]
```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
```
Comment (pluhin) [inline Nmaksimchicov/3.HTML/1.html:1]: label need to bound with particular form unit with attribute for, not form, and a value of this for should be unic on this page

File: Nmaksimchicov/2.HTML/Месяц.html [hunk 13]
```diff
@@ -0,0 +1,36 @@
...
+<html>
+	<head>
+	<META charset="utf-8">
+	<title>My month</title>
+</head>
+<body>
+<table width="200" border="7">
+		<col width="20"><col width="20"><col width="20"><col width="20"><col width="20"><col width="20">
+		<tr>	
+								<th valign="middle" colspan="7">April</th>
```
Comment (pluhin) [inline Nmaksimchicov/2.HTML/Месяц.html]: valign already has this value

File: Nmaksimchicov/2.HTML/Задание 3.html [hunk 14]
```diff
@@ -0,0 +1,73 @@
...
+def DoTwo(num1,num2): 
+	print '%d + %d = %d ' % (num1,num2,num1+num2) 
+	print '%d * %d = %d ' % (num1,num2,num1*num2) 
+	print '%d - %d = %d ' % (num1,num2,num1-num2) 
+	print '\n'
+DoTwo(1,2)
+DoTwo(1,4)
+DoTwo(10,5)</pre> 
+<p>Как видите, здесь написано меньше кода — 8 строк вместо 12. Если нам нужно будет что-либо поменять в функции, мы сможем сделать это, не создавая проблем для нашей программы. Чтобы вызвать функцию, нужно написать её имя и параметры.</p> 
+	<table width="1420" border="0">
```
Comment (pluhin) [inline Nmaksimchicov/2.HTML/Задание 3.html]: Could you please use % here, need to be more adaptive, check all  your python files

File: Nmaksimchicov/1.HTML/5.html [hunk 15]
```diff
@@ -0,0 +1,71 @@
...
+<h3>Магазин работает:</h3>
+<p>c 08<sup><ins>30</ins></sup> до 18<sup><ins>30</ins></sup></p>
+<p>обед с 13<sup><ins>00</ins></sup> до 14<sup><ins>00</ins></sup></p>
+<p>выходной: <ins>суббота</ins>, <ins>воскресенье</ins></p>
+<hr width="100%" color="##000000" align="left" size="1">
+<h3>Вот такой текст</h3>
+<p>Э<sub>т</sub> о <sub>п</sub>р<sub>о</sub>с<sub>т</sub>о <sub>в</sub>е<sub>с</sub>ё<sub>л</sub>ы<sub>й</sub> т<sub>е</sub>к<sub>с</sub>т</p>
+<hr width="100%" color="##000000" align="left" size="1">
+<h2>Акроним и курсив</h2>
+<p>ТФ: Y(<em>x</em>)=e­<sup><em>-iπx</em></sup>+log<sub><em>x</em></sub><sup>4</sup></p>
```
Comment (pluhin) [inline Nmaksimchicov/1.HTML/5.html:65]: replace pi to mnemonic

**Review by pluhin — CHANGES_REQUESTED** [review]

**Comment by pluhin** [comment]
Any changes?

File: Nmaksimchicov/2.HTML/Задание 3.html (hunk 14 above)
Comment (GRAFOLAF) [reply Nmaksimchicov/2.HTML/Задание 3.html]: Can you please explain a bit more in details cos I don't really get the problem here ? thx =)

Comment (pluhin) [reply Nmaksimchicov/2.HTML/Задание 3.html]: You have 1420 - as width, I have large screen and your menu to sort, you need to make pages more adaptive

**Pull request merged**